components/image/image.css

```diff
@@ -0,0 +1,28 @@
+.img-rsp {
```

Comment:
i will suggest to keep class name more understandable like .img-responsive

Reply:
Thanks for the suggestion, done now!